src/languages/en.js

```diff
@@ -1157,12 +1157,19 @@ export default {
     newTaskPage: {
         task: 'Task',
         assignTask: 'Assign task',
-        title: 'Title',
+        assignee: 'Assignee',
+        assigneeError: 'There was an error assigning this task, please try another assignee',
+        confirmTask: 'Confirm task',
+        confirmError: 'Please enter a title and select a share destination',
+        title: 'Task title',
```

Comment:
This might be incorrect in the mockup, but this should just be 'Title' since it's already implied we're in a task